### PR TITLE
PHP-CS-Fixer fix all the things!

### PR DESCRIPTION
**Pull request by bmansell**
- Added `friendsofphp/php-cs-fixer` **version 2.0.0** as a dev dependency along with a `.php_cs` config file
  - Following PSR2 and Symfony standards
  - Chose 2.0.0 because the package itself and its dependencies (`symfony) are supported by our project's php version (5.6.30)
- Ran the fixer on the project
- Removed `Class`/`Interface`/`Trait` block comments
- Fixed `@var` block comments to be one liners
  - If you're curious, used the following regex to find instances of the two above points: 
    1. Class blocks: `\/\*\*\n *\* @var \w+\n *\*\/`
    2. Var blocks: `\/\*\*\n *\* Class \w.+\n *\*\/`

### CODE REVIEW

File: src/Services/Client.php [hunk 1]
```diff
@@ -95,19 +96,18 @@ public function getRedirectLocation($path, $params = [])
...
-     * @param bool $frontendApi
+     * @param bool   $frontendApi
      *
      * If password is set it will auth to /password before it does anything. Useful for frontend calls.
      * Cookies is an array of SetCookie objects. See the Cart service for an example.
      *
      * @return array
      */
-    public function post($path, $params = [], $body, array $cookies = [], $password = null, $frontendApi = false)
+    public function post($path, $params, $body, array $cookies = [], $password = null, $frontendApi = false)
```
Comment (bmansell) [inline src/Services/Client.php]: This change is the only one that concerns me. I looked at all usages of this function however and we are passing in an empty array in all instances.

File: src/Services/Client.php [hunk 2]
```diff
@@ -129,7 +129,7 @@ public function post($path, $params = [], $body, array $cookies = [], $password
      *
      * @return array
      */
-    public function put($path, $params = [], $body)
+    public function put($path, $params, $body)
```
Comment (bmansell) [inline src/Services/Client.php]: Same consideration and conclusion as my comment above.

Comment (rickywiens) [reply src/Services/Client.php]: This makes sense. If we wanted to keep this, we should also create a default value for the $body variable. I'm not sure which I prefer, they both seem reasonable.

Comment (theRTC204) [reply src/Services/Client.php]: PHP requires that defaulted function arguments be placed after all non-defaulted arguments. Based on that alone, the defaulted `$params` would never have worked before this change.

This seems perfectly safe to me.

**Pull request merged**